src/NCronJob/Registry/JobRegistry.cs

```diff
@@ -57,7 +57,7 @@ public void RemoveByType(Type type)
         }
     }
 
-    public JobDefinition AddDynamicJob(
+    public static JobDefinition BuildDynamicJob(
```

Comment:
nitpick: `Create` would be a more ".NET"y prefix

Reply:
Fixed